subsys/emul/spi/emul_bmi160.c

```diff
@@ -2,6 +2,9 @@
  * Copyright 2020 Google LLC
  *
  * SPDX-License-Identifier: Apache-2.0
+ *
+ * Emulator for the Boche BMI160 accelerometer / gyro. This supports basic
+ * init and reading of canned samples. It supports both I2C and SPI buses.
```

Comment:
The files location doesn't make sense anymore if this file support both SPI and I2C.  Can you put this in subsys/emul/sensors/emul_bmi160.c instead?

Reply:
OK done